modules/study/src/main/StudyApi.scala

```diff
@@ -207,6 +215,10 @@ final class StudyApi(
 
   def kick(studyId: Study.Id, userId: User.ID) = sequenceStudy(studyId) { study =>
     study.isMember(userId) ?? {
```

Comment:
That line was written by you :P Anyway I tried changing it, and I got a compiler error because it expect a Funit, so I guess I'm keeping the ?? here.